src/Mount.cpp

```diff
@@ -1663,9 +1671,9 @@ void Mount::setSpeed(StepperAxis which, float speedDegsPerSec)
         }
     #elif AZ_DRIVER_TYPE == DRIVER_TYPE_A4988_GENERIC || AZ_DRIVER_TYPE == DRIVER_TYPE_TMC2209_STANDALONE                                  \
         || AZ_DRIVER_TYPE == DRIVER_TYPE_TMC2209_UART
-        float stepsPerSec = speedDegsPerSec * _stepsPerAZDegree;  // deg/sec * u-steps/deg = u-steps/sec
-        LOGV3(DEBUG_STEPPERS, F("STEP-setSpeed: Set AZ speed %f degs/s, which is %f steps/s"), speedDegsPerSec, stepsPerSec);
-        _stepperAZ->setSpeed(stepsPerSec);
+            float stepsPerSec = speedDegsPerSec * _stepsPerAZDegree;  // deg/sec * u-steps/deg = u-steps/sec
```

Comment:
Why the indents here? Was it required for `clang-format`? Seems weird

Reply:
Yeah, didn't pass the clang test otherwise

Reply:
Let me try again.

Reply:
Looks like it took.... so extra indentation it doesn't seem to check...